tensorrt_llm/inputs/multimodal.py

```
@@ -185,39 +198,55 @@ def __post_init__(self):
+                self.num_mm_tokens_in_chunk = (chunk_end_idx -
+                                               self.num_unseen_mm_tokens)
+                remainder = len(p) - chunk_end_idx
+            else:
+                # Fallback: range-based computation using pos + length.
+                # This is correct when MM tokens within each chunk are contiguous,
+                # but may overcount when text gaps exist within a chunk.
+                self.num_unseen_mm_tokens = 0
+                self.num_mm_tokens_in_chunk = 0
+                remainder = 0
+                for pos, length in zip(self.mm_token_positions,
+                                       self.mm_token_lengths):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)